### ISSUE TITLE
Cochlearia acaulis / Ionopsidium acaule

### ISSUE THREAD

**Issue by japonicus** [BSBI/mapmate-taxon-requests#28]
MapMate has separate entries for Ionopsidium acaule (v04331hb) and Cochlearia acaulis (v39531zj)

I think I. acaule should be set as a synonym of C. acaulis

I'm unsure if we should use Ionopsidium or Jonopsidium for the genus (Stace uses 'J').

**Comment by sacrevert** [comment]
Yes, looks like they are synonymous. Looks like Jonopsidium is the correct spelling. As far as I can tell this is Reichenbach's original publication of the name:
https://books.google.co.uk/books?id=8HEFAAAAQAAJ&pg=PA26&lpg=PA26&dq=Iconographia+botanica+seu+plantae+criticae+jonopsidium&source=bl&ots=IOTP1Rq4Yb&sig=M9esH18ZCF7m-L12ba_ucrhr-3o&hl=en&sa=X&ved=0ahUKEwjyvt_D85DZAhXoLcAKHS3DDJ8Q6AEIZjAN#v=onepage&q=Iconographia%20botanica%20seu%20plantae%20criticae%20jonopsidium&f=false

**Comment by sacrevert** [comment]
The confusion comes from the fact that De Candolle put Cochlearia acaulis into a section called 'Ionopsidium' https://books.google.co.uk/books?id=qmNsPMi7pBcC&pg=PA174&lpg=PA174&dq=de+candolle+prodromus+ionopsidium&source=bl&ots=NZtvq8q3s_&sig=iJz9LVuW1egSt88gwB8xkQPe_-g&hl=en&sa=X&ved=0ahUKEwjaxojv9JDZAhVGW8AKHU3ID1EQ6AEIPDAD#v=onepage&q=de%20candolle%20prodromus%20ionopsidium&f=false

I have no idea whether Reichenbach's, no doubt conscious, change of spelling stands. I suppose it depends on whether De Candolle's section legitimately has priority, and whether Reichenbach's genus was actually supposed to be De Candolle's section at another level, or whether Reichenbach just felt like it was a good name. ITIS considers that Reichenbach was wrong to change the spelling (https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=23152#null).

#############################################################################
Actually, considering all of the above, it looks like Ionopsidium is correct. Reichenbach notes that acaulis is the only species in that section in De Candolle's Prodromus, so I suppose it is clear that Reichenbach is just changing the section to the genus level, in which case I suppose he should have followed De Candolle's spelling :)

**Comment by sacrevert** [comment]
Mabberley agrees with this assessment: https://books.google.co.uk/books?id=ZVdoUPtqKCIC&printsec=frontcover&dq=mabberley&hl=en&sa=X&ved=0ahUKEwjnotjx7pDZAhXKAsAKHWMjCmEQ6AEIJzAA#v=snippet&q=jonopsidium&f=false
